cypress/integration/queryBuilder.spec.js

```diff
@@ -30,4 +30,18 @@ describe('test query builder', () => {
     cy.get('[data-cy="applyFiltersButton"]').click();
     cy.get('[data-cy=appliedFilterList-ancestry_specific_meta_analysis]', {}).should('be.visible');
   });
+
+  it('adds/removes readers', () => {
+    // selects the share button in the sidebar
+    cy.get('div.MuiButtonBase-root:nth-child(3) > svg:nth-child(1)').click();
+    cy.get('[data-cy=enterEmailBox]').type('mkerwin.dev@gmail.com');
+    cy.get('[data-cy=inviteButton]').click();
+    cy.get('[data-cy=readers]')
+      .contains('mkerwin.dev@gmail.com')
+      .should('be.visible');
```

Comment:
How about a test to type in a comma between emails and see if you get two entries?

Reply:
done!